src/components/PageNotFound/PageNotFound.jsx

```diff
@@ -0,0 +1,19 @@
+import React from 'react';
+import { Link } from 'react-router-dom';
+
+// Styles
+import './PageNotFound.scss';
+
+export default () => {
+  return (
+    <div className="pagenotfound-container">
+      <h1 className="pagenotfound-container__head">Page Not Found</h1>
+      <h4 className="pagenotfound-container__text">
+        Oops! The page you are looking for has been removed or relocated.
+      </h4>
+      <div className="pagenotfound-container__button">
+        <Link to="/">Go Back</Link>
```

Comment:
You're not going back to be precise. It's homepage always.
Change this to "Go Back to Homepage" or "Go to Homepage"